.yamllint

```diff
@@ -1,11 +1,21 @@
+---
+# https://github.com/ansible/galaxy/blob/devel/galaxy/importer/linters/yamllint.yaml
```

Comment:
Much better solution than me adding spaces. I learned this today, thank you.

Reply:
ha, nice! Yeah, I noticed this from https://galaxy.ansible.com/docs/contributing/content_scoring.html#linters-used actually. Seems sane to me.